### DISCUSSION TITLE
a question about deepflame install

### DISCUSSION THREAD

**Discussion by MacCary** [deepmodeling/deepmd-kit#2460] [Q&A] [answered]
Hello, I'm sorry to bother you. I have some questions to ask you. I really hope you can answer them. I had some problems installing deepflame. The first question is: there is no envt/deepflame/lib/include/etc this folder. The second is that there is no error during the compilation process, but it often appears that could not find XXX file or dictonry, one of the files that could not be found is the fmt/format.h header file, and others (I do not know whether these header files are in the folder I just said). As a result, four of deepflame's built-in solvers did not work properly. I'm sorry to send you a private message in my busy schedule. I hope you can answer it. Thank you very much.

**Comment by njzjz** [comment, marked as answer]
Hi, please ask questions in the deep-flame repository: https://github.com/deepmodeling/deepflame-dev
